alts/src/generated/main/grpc/io/grpc/alts/internal/HandshakerServiceGrpc.java

```diff
@@ -30,6 +30,12 @@ private HandshakerServiceGrpc() {}
   private static volatile io.grpc.MethodDescriptor<io.grpc.alts.internal.Handshaker.HandshakerReq,
       io.grpc.alts.internal.Handshaker.HandshakerResp> getDoHandshakeMethod;
 
+  @io.grpc.stub.annotations.RpcMethod(
+      fullServiceName = "grpc.gcp.HandshakerService",
+      methodName = "DoHandshake",
+      inputType = io.grpc.alts.internal.Handshaker.HandshakerReq.class,
```

Comment:
personal preference: I like `requestType` and `responseType`.  Input and output don't match any of our other verbiage.

Reply:
Good point. Yeah, we should fix that.